AutoSteamApp/Helpers/StaticHelpers.cs

```diff
@@ -0,0 +1,179 @@
+        }
+
+        /// <summary>
+        /// Logs the current configuration settings to the user
+        /// </summary>
+        public static void DisplayConfig()
+        {
+            Log.Message("Configuration Settings:");
+            Log.Message("\tApplication Version: " + ConfigurationReader.ApplicationVersion);
+            Log.Message("\tIs Debug: " + ConfigurationReader.IsDebug);
+
+            if (!ConfigurationReader.ConfigLoadedProperly)
+            {
+                Log.Warning("Config file was unable to load.");
+                return;
+            }
+            Log.Message("\tIs Azerty: " + ConfigurationReader.IsAzerty);
+            Log.Message("\tLog File: " + (ConfigurationReader.LogFile ?? "Not supplied") );
+            Log.Message("\tRandom Run: " + ConfigurationReader.RandomRun);
+            Log.Message("\tInput Delay on Random Run: " + ConfigurationReader.RandomInputDelay);
+            Log.Message("\tCutscene skip key: " + ConfigurationReader.KeyCutsceneSkip.ToString());
+            Log.Message("\tCommon Reward Success Rate: " + ConfigurationReader.CommonSuccessRate.ToString("#.##"));
+            Log.Message("\tRare Reward Success Rate: " + ConfigurationReader.RareSuccessRate.ToString("#.##"));
+            Log.Message("\tMaximum Wait for Determining Slot: " + ConfigurationReader.MaxTimeSlotNumberSeconds);
+            Log.Message("");
+        }
+
+
+        /// <summary>
+        /// Sets the proper log methods depending on the configuration found in the .config file
+        /// </summary>
+        public static void SetLogger()
+        {
+            // Initialize the different log types
+            Log.LogTypes LoggingTypes = Log.LogTypes.Message | Log.LogTypes.Warning;
+
+            // If we're in debug mode, heighten the logging which takes place
+            if (ConfigurationReader.IsDebug)
+            {
+                LoggingTypes |= Log.LogTypes.Debug | Log.LogTypes.Exception | Log.LogTypes.Error;
+
+                // Check if we need to write logs to a file
+                string logFile = ConfigurationReader.LogFile;
+                if (!string.IsNullOrEmpty(logFile))
+                {
+                    // Try to create a stream using the log file
+                    try
+                    {
+                        FileStream logStream = File.Create(logFile);
+                        Log.SetStream(logStream, true, LoggingTypes);
+                        return;
+                    }
+                    // In the event of failure, log to console instead
+                    catch (Exception e)
+                    {
+                        Log.SetStream(Console.OpenStandardOutput(), false, LoggingTypes);
+                        Log.Exception(e, logFile);
+                        return;
+                    }
+                }
+                // If no log file is specified, log to console instead
```

Comment:
Should always follow the `empty line` rule after a `}` except for when there are multiple `}`